flutter_local_notifications/example/integration_test/flutter_local_notifications_test.dart

```diff
@@ -34,6 +34,35 @@ void main() {
           .initialize(initializationSettings);
       expect(initialised, isTrue);
     });
+
+    testWidgets('initialize should throw an ArgumentError',
+        (WidgetTester tester) async {
+      const InitializationSettings initializationSettings =
+          InitializationSettings();
+      try {
+        await flutterLocalNotificationsPlugin
+            .initialize(initializationSettings);
+        // ignore: avoid_catches_without_on_clauses
+      } catch (e) {
+        expect(e, isArgumentError);
+        if (Platform.isAndroid) {
+          expect(e.message,
+              'Android settings must be set when targeting Android platform');
+        }
+        if (Platform.isIOS) {
+          expect(e.message,
+              'iOS settings must be set when targeting iOS platform');
+        }
+        if (Platform.isLinux) {
+          expect(e.message,
+              'Linux settings must be set when targeting Linux platform');
+        }
+        if (Platform.isMacOS) {
+          expect(e.message,
```

Comment:
casing issue, the platform itself is referred to as "macOS" with a lowercase "m"

Reply:
Not sure what you want to say here https://api.flutter.dev/flutter/package-platform_platform/Platform/isMacOS.html

Reply:
For convenience, I'll reply to both of your responses related this topic here. First off is you are linking to the wrong doc. What you were using came from dart:io, see https://api.flutter.dev/flutter/dart-io/Platform/isMacOS.html. The rule of having first letter in uppercase for the first word in a sentence doesn't apply here as Apple has gone with lowercase "m" as part of they're branding. if you click through the link you will see that it in turn has a link to Wikipedia with macOS as the first word with lowercase "m" too. If you do a search for "macOS" then you should find other examples, particularly articles written that have it as the first word and respect the casing Apple has chosen due to branding too.

In other words, this will need lowercase "m" for all instances where it is mentioned. Note that even in your original PR you had "iOS" even when it was the first word. This is correct though not consistent with what you thought/point around casing rules

Reply:
fixed